src/Stoopy.tsx

```diff
@@ -65,21 +68,39 @@ const Component = ({
   const NextButton = components.NextButton || defaultLayout.NextButton;
   const Loading = components.Loading || defaultLayout.Loading;
 
+  const [transition, setTransition] = useState(true)
+
   return saving ? (
     <Loading />
   ) : (
     <>
       {field && (
         <form
+          key="form"
           onSubmit={e => {
             e.preventDefault();
             // setLoading(true)
-            onNext(filteredOutput);
+            // onNext(filteredOutput);
```

Comment:
Was this commented for technical issues or  UX choice?

Reply:
Opss, I forgot. Actually, I didn't understand why onNext was on both events (onSubmit and onClick). Why?